03_flow_MEFL/exercises.m

```diff
@@ -67,11 +71,20 @@
 colorpairfiles{1} = {channels{1}, channels{3}, channels{2}, [colordata '2012-03-12_mkate_EBFP2_EYFP_P3.fcs']};
 colorpairfiles{2} = {channels{2}, channels{3}, channels{1}, [colordata '2012-03-12_mkate_EBFP2_EYFP_P3.fcs']};
 colorpairfiles{3} = {channels{2}, channels{1}, channels{3}, [colordata '2012-03-12_mkate_EBFP2_EYFP_P3.fcs']};
-
 CM = ColorModel(beadfile, blankfile, channels, colorfiles, colorpairfiles);
-CM = set_ERF_channel_name(CM, 'FITC-A'); % Name the channel we'll use for ERF units
+CM=set_translation_plot(CM, true);
+CM=set_noise_plot(CM, true);
+
+TASBEConfig.set('beads.beadModel','SpheroTech RCP-30-5A'); % Entry from BeadCatalog.xls matching your beads
+TASBEConfig.set('beads.beadBatch','Lot AA01, AA02, AA03, AA04, AB01, AB02, AC01, GAA01-R'); % Entry from BeadCatalog.xls containing your lot
+% Can also set bead channel if, for some reason, you don't want to use fluorescein as standard
+% This defaults to FITC as it is strongly recommended to use fluorescein standards.
+% TASBEConfig.set('beadChannel','FITC');
+
+CM = set_ERF_channel_name(CM, 'FITC-A'); % We'll explain this in the next exercise
```

Comment:
Delete comment about explaining in the next exercise, since this is that exercise?

Reply:
done